wagtail/wagtailadmin/static_src/wagtailadmin/js/core.js

```diff
@@ -58,7 +58,7 @@ var dirtyFormCheckIsActive = true;
 function enableDirtyFormCheck(formSelector, options) {
     var $form = $(formSelector);
     var $ignoredButtons = $form.find(
-        options.ignoredButtonsSelector || 'input[type="submit"],button[type="submit"]'
+        options.ignoredButtonsSelector || 'input[type="submit"],button[type="submit"],button.action-preview'
```

Comment:
It would be better to add this as an `ignoredButtonsSelector` parameter on the call to enableDirtyFormCheck (https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailadmin/templates/wagtailadmin/pages/create.html#L81-L89 and https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailadmin/templates/wagtailadmin/pages/edit.html#L115-L124) - `enableDirtyFormCheck` is intended to be a generic function, not tied to any particular form.